### ISSUE TITLE
Provide version information

### ISSUE THREAD

**Issue by biolauri** [iamkarlson/keepass-quicksearch#3]
When KeePass checks for updates, the entry of this plugin shows status _Unknown._ and installed version _0.0.0.1_. I think, it populates this field with the `AssemblyFileVersion` in _QuickSearch_Properties_AssemblyInfo.cs_.

Additionally, I would appreciate it, if you provide the latest version via a (GitHub-)URL, so the Update Check can test whether the latest version is installed or not. Like [jnko/SimpleDatabaseBackup](https://github.com/jnko/SimpleDatabaseBackup) (_SimpleDatabaseBackupVERSION.txt_), you could use a simple *.txt-file which contains the latest version and will be updated after every major commit.

Besides, thank you for the great Plugin! :+1:

**Comment by iamkarlson** [comment]
Check [link](https://github.com/iamkarlson/keepass-quicksearch/blob/master/quicksearchVERSION.txt)

**Comment by iamkarlson** [comment]
Check release [2.27](https://github.com/iamkarlson/keepass-quicksearch/releases/tag/2.27)

**Comment by biolauri** [comment]
Great! Thank you very much! Just updated and tested the installed version and it displays the correct string.

In your link I saw, you added the version file and I just realised, I forgot to mention the settings implementation of the integrated update check. Besides this file, KeyPass has to know, where to check this: I think, the example plug in does this with a [simple function in the main *.cs-file](https://github.com/jnko/SimpleDatabaseBackup/blob/master/SimpleDatabaseBackup/SimpleDatabaseBackup.cs#L37).
Sorry about that!

Maybe, I'll manage to do a pull request about this later today, but I won't promise as my schedule is a bit tight.
